apps/editing-toolkit/editing-toolkit-plugin/help-center/src/config.js

```diff
@@ -4,7 +4,7 @@ window.configData = {
 	google_analytics_key: 'UA-10673494-15',
 	client_slug: 'browser',
 	twemoji_cdn_url: 'https://s0.wp.com/wp-content/mu-plugins/wpcom-smileys/twemoji/2/',
-	happychat_url: 'https://happychat.io/customer',
+	happychat_url: 'https://happychat-io-staging.go-vip.co/customer',
```

Comment:
Since we are getting ready to officially merge, just pointing out these things that we may want to change back.

Reply:
Let's leave it like this in this PR, I'll revert this https://github.com/Automattic/wp-calypso/pull/66360 to have the right env at the right time.